docs/source/platform/operation-registry.md

```diff
@@ -290,6 +290,50 @@ const server = new ApolloServer({
 });
 ```
 
+## Migrating from 0.1
```

Comment:
Since we're going from alpha to alpha, i think it would be more clear to offer 0.1-alpha.4